code/game/objects/items/weapons/implants/implant_mindshield.dm

```diff
@@ -25,15 +25,15 @@
 			to_chat(target, "<span class='warning'>You feel the corporate tendrils of Nanotrasen try to invade your mind!</span>")
```

Comment:
might want to edit this line aswell?

Reply:
Indeed. That's what I mentioned in the PR. Though I'd need suggestions on how to change that line as it's a bit more niche than the other line of text. And hard to make it sound non magic-y and non loyalty-y.

The only thing that really comes to mind is something like; "You feel something trying to remove the corrupting thoughts from your mind.". Though I'm not sure if that feels too magic-y or not. If that's fine I can go with that.

Reply:
I feel like it's sorta fine, it's not physical corporate tendrils it's more like the cultists thoughts on nanotrasen? I can't really think of a good replacement.